src/sentry/transactionHub.ts

```diff
@@ -0,0 +1,37 @@
+import * as Sentry from '@sentry/react-native'
+import { Transaction, TransactionContext } from '@sentry/types'
+
+// @ts-ignore
+export class TransactionHub {
+  private static instance: TransactionHub
+  transactions: Array<Transaction>
+
+  private constructor() {
+    this.transactions = []
+  }
+
+  public static getInstance(): TransactionHub {
```

Comment:
hum, i'm wondering if we need this singleton implementation here. a few reasons i think this
- it's not a pattern that we currently use in the project. i generally don't have strong feelings about code patterns because they all have pros and cons, but i feel strongly about consistency because it makes code more predictable/readable/maintainable. we have some similar classes that act like singletons in the code already, like ValoraAnalytics and Logger. I would advocate for adopting the same approach as the Logger (i.e. exporting and consuming the class in the same way)
- i think JS has the singleton pattern baked in, with object literals. this article might be interesting, i'm thinking we could simplify a lot if we aren't so verbose about the singleton while still getting the benefits of a singleton https://codeburst.io/javascript-global-variables-vs-singletons-d825fcab75f9#:~:text=Singleton%20is%20an%20object%20which,the%20reference%20to%20that%20object.&text=Technically%20speaking%2C%20object%20literals%20in%20JavaScript%20are%20singletons.

Reply:
Changed to an object literal in https://github.com/valora-inc/wallet/pull/2728/commits/485a85f463b1b291e2caace4f93e752b9cce4ae8!